src/Microsoft.Data.Sqlite/project.json

```diff
@@ -21,7 +21,7 @@
     ]
   },
   "dependencies": {
-    "SQLite": "3.12.2",
+    "SQLitePCLRaw.bundle_e_sqlite3": "1.1.0-pre20160928150051",
```

Comment:
Ah, yes, the discussion over in item #148 mentions some of the same techniques that I've needed to use.